### ISSUE TITLE
New chat

### ISSUE THREAD

**Issue by Lordwaru** [4a/fv-front#12]
Are we still doing a new chat?

I have no job and im bored as fuck.

**Comment by 4a** [comment]
yeah we can start it. all i've done so far is think of fun features

**Comment by JoeStaines** [comment]
Are you moving away from chatango yet?

**Comment by 4a** [comment]
yeah i think we should just make our own chat with some nodejs websocket thing

i was thinking we could connect the chat to the list of streams youre watching and do funny stuff with it
